src/appleseed/foundation/core/appleseed.cpp

```diff
@@ -30,6 +30,9 @@
 // Interface header.
 #include "appleseed.h"
 
+// appleseed.renderer headers.
```

Comment:
What about moving this to renderer/core for example. It's not really related to foundation.